src/Http/Http.Extensions/test/RequestDelegateFactoryTests.cs

```diff
@@ -226,6 +226,11 @@ private static void TestOptionalNullable(HttpContext httpContext, int? value = 4
         httpContext.Items.Add("input", value);
     }
 
+    private static void TestOptionalNullableNull(HttpContext httpContext, double? value = null)
+    {
+        httpContext.Items.Add("input", value);
```

Comment:
```suggestion
        httpContext.Items.Add("input", value ?? "null");
```

The `ItemsDictionary` is unusual in that it returns `null` for missing values, so it's impossible to know from a test if this line executed or not.

https://github.com/dotnet/aspnetcore/blob/e001dc01a152806a457eb9da15dc102107451dee/src/Http/Http/src/Internal/ItemsDictionary.cs#L23-L33

Reply:
Many thanks for spotting that @halter73. I have updated the Acerby:main branch based on a slightly updated version your suggestion of using "null".